### PR TITLE
Improve logs on publish

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related #20

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@noahlitvin  It's a little different with what you said in 
https://github.com/usecannon/cannon/issues/20#issuecomment-1596171529

because You said about logging some parameters that I didn't know hot can I have them before publishing package, anyway I think better way is to just create my PR and you leave your feedback here
Sorry I know it's a tiny PR and you might can implement it faster than explaining it

File: packages/cli/src/commands/publish.ts [hunk 1]
```diff
@@ -42,18 +43,27 @@ export async function publish({
...
     console.log(blueBright('publishing remote ipfs package', packageRef));
+    console.log(
+      blueBright(
+        'Uploading the following Cannon package data to',
+        cliSettings.publishIpfsUrl,
+        'Tags',
+        tags,
+        'Variant',
+        `${chainId!}-${preset!}`,
+        'QM Hash'
```
Comment (dbeal-eth) [inline packages/cli/src/commands/publish.ts:63]: no qm hash is included after

**Review by dbeal-eth — APPROVED** [review]
thanks for this

File: packages/cli/src/commands/publish.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply packages/cli/src/commands/publish.ts]: Thanks, Done.

**Pull request merged**